sdk/core/azure-core/CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 ### Bug Fixes
 
 - Make sure to rewind the body stream at the start of each request retry attempt, including the first.
+- Fix `Azure::Context` to support unique_ptr.
```

Comment:
Was this broken in the previous release or is this a recent issue? If it was working before, we don't need a CL.

Reply:
broken since https://github.com/Azure/azure-sdk-for-cpp/pull/444/files  (Ag 2020)

Reply:
No one used it, so, no CL?